stubs/Flask-SQLAlchemy/flask_sqlalchemy/model.pyi

```diff
@@ -1,7 +1,11 @@
 from re import Pattern
-from typing import Any
+from typing import Any, Generic, TypeVar
 
+from sqlalchemy import Table
 from sqlalchemy.ext.declarative import DeclarativeMeta
+from sqlalchemy.orm import Query
+
+_Model = TypeVar("_Model")
```

Comment:
Style nit: I'd prefer we called this `TypeVar` `_ModelT`; the `T` suffix indicates unambiguously that it's a `TypeVar` rather than a `TypeAlias`.

```suggestion
_ModelT = TypeVar("_ModelT")
```

Reply:
Sorry about that. Makes sense.

Reply:
No need to be sorry! :)